packages/manager/apps/dedicated/client/app/components/dedicated-cloud/datacenter/host/order/dedicatedCloud-datacenter-host-order.controller.js

```diff
@@ -71,38 +63,61 @@ export default class {
               datacenterId: this.datacenterId,
             },
           },
-        ).then((profiles) => {
-          const result = [];
+        ).then((profiles) =>
+          this.$http
+            .get('/products/partners/plans', {
+              rootPath: 'apiv6',
+              params: {
+                ovhSubsidiary: this.user.ovhSubsidiary,
+                publicPlanCode: profiles.map((p) => p.name),
+              },
+            })
+            .then((response) => {
+              const plans = response.data;
 
-          angular.forEach(offers, (offer) => {
-            const profile = filter(profiles, { name: offer.planCode });
-            if (size(profile) === 1) {
-              set(offer, 'profile', head(profile));
-              result.push(offer);
-            }
-          });
+              const sortedResult = offers
+                .filter((offer) =>
+                  profiles.find((p) => p.name === offer.planCode),
+                )
+                .map((offer) => {
+                  const plan = plans.find(
+                    (p) => p.publicPlanCode === offer.planCode,
+                  )?.partnerPlan;
+                  const { prices } = offer;
+                  // If available replace pricing text and value with partner ones
+                  if (plan) {
+                    prices[0].price.text = plan.pricings[0].formattedPrice;
+                    prices[0].price.value = plan.pricings[0].price / 100000000;
+                  }
```

Comment:
I agree with you but after checking with Dan from order team, they are not able to handle it in /order/cartServiceOption/privateCloud/ as it would introduce prices discrepancies for other products between the manager and the order sites